### PR TITLE
Add tokens to config

### PR DESCRIPTION
**Pull request by seanwiseman**

### CODE REVIEW

File: salt/profiles/config/srv-profiles-app.cfg [hunk 1]
```diff
@@ -8,6 +8,8 @@ authorize_uri = {{ pillar.profiles.orcid.authorize_uri }}
 token_uri = {{ pillar.profiles.orcid.token_uri }}
 client_id = {{ pillar.profiles.orcid.client_id }}
 client_secret = {{ pillar.profiles.orcid.client_secret }}
+public_access_token = {{ pillar.profiles.orcid.public_access_token }}
```
Comment (thewilkybarkid) [inline salt/profiles/config/srv-profiles-app.cfg]: `read_public_access_token` (to match the scope)?

File: salt/profiles/config/srv-profiles-app.cfg [hunk 2]
```diff
@@ -8,6 +8,8 @@ authorize_uri = {{ pillar.profiles.orcid.authorize_uri }}
...
+webhook_token = {{ pillar.profiles.orcid.webhook_token }}
```
[the remark this answers is not in the document]
Comment (seanwiseman) [reply salt/profiles/config/srv-profiles-app.cfg]: Done.

File: salt/profiles/config/srv-profiles-app.cfg (hunk 1 above)
Comment (seanwiseman) [reply salt/profiles/config/srv-profiles-app.cfg]: Done.

File: salt/pillar/profiles.sls [hunk 3]
```diff
@@ -20,6 +22,8 @@ profiles:
         port: 5432
     logging:
         level: DEBUG
+    orcid_dummy:
```
Comment (giorgiosironi) [inline salt/pillar/profiles.sls]: this has been removed in favor of `pillar.elife.php_dummies.orcid_dummy`, should not be necessary

**Pull request merged**